feature/gribi/ate_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -160,12 +164,29 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) *ondatra.ATETopology {
 	return top
 }
 
+// Configure Network instance
+func configNetworkInstance(t *testing.T, dut *ondatra.DUTDevice, vrfname string, intfname ...string) {
+	d := &oc.Root{}
+	ni := d.GetOrCreateNetworkInstance(vrfname)
+	ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	if len(intfname) != 0 {
+		intf := ni.GetOrCreateInterface(intfname[0])
+		intf.Interface = ygot.String(intfname[0])
+	}
+	gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(vrfname).Config(), ni)
+}
+
 // configureDUT configures port1, port2, port3 and port4 on the DUT.
 func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	d := gnmi.OC()
 
 	p1 := dut.Port(t, "port1")
-	gnmi.Replace(t, dut, d.Interface(p1.Name()).Config(), dutPort1.NewOCInterface(p1.Name()))
```

Comment:
What is the reason for this change?

Reply:
> Actually I'm fine with removing this line. Is it a concern ? @LimeHat
Essentially the question is "why we are replacing L168 with L189"
I.e. the typical order of operations is 1) configure interface (usually with a replace but I'm not against an update either) 2) assign it to a network instance

Not sure why both changes were needed here?

Reply:
This is the order of operation done here 
1) Replace on port1 with vrf1 (assign it to a network instance)
2) Update on port1 with v4/v6 ip and other configs . Doing a replace instead of this update will remove previous network instance configs

Reply:
@agvarghe-cisco 
Could you please implement this in the following way:
1) configure physical interface (gnmi Replace)
2) configure a subinterface w/o IP configuration
3) assign the subinterface to a vrf
4) update subinterface with IP configuration (gnmi update)
(you can combine multiple steps in a single gnmi call, e.g. 1-2 or 2-3 or 1-3)
An example of such config order can be found in test case RT-3.2

cc @xw-g

Reply:
Looks good now. Thanks